src/Workspaces/Core/Portable/FindSymbols/FindReferences/DependentTypeFinder.cs

```diff
@@ -358,8 +360,7 @@ private static void AddRange(SymbolSet foundTypes, SymbolSet currentTypes, Func<
             // that actually supports compilations.
```

Comment:
Either this comment is wrong, or your removal of the `Where` is wrong, but I have no idea which

Reply:
Yes. That's fair.